controllers/idler/idler_controller_test.go

```diff
@@ -208,7 +210,7 @@ func TestEnsureIdling(t *testing.T) {
 				// Still tracking all pods. Even deleted ones.
 				memberoperatortest.AssertThatIdler(t, idler.Name, cl).
 					TracksPods(podsTooEarlyToKill.allPods).
-					HasConditions(memberoperatortest.Running())
+					HasConditions(memberoperatortest.Running(), memberoperatortest.IdlerNotificationCreated())
```

Comment:
we should also make sure that the Notification CR actually exists in the host client when it should, and that it doesn't when it should not.